pages/_app.tsx

```diff
@@ -23,22 +25,19 @@ export default function App({ Component, pageProps, router }: AppProps) {
   if (router.pathname.startsWith('/code')) {
     return <Component {...pageProps} />
   }
-
-  if (
-    router.pathname.startsWith('/realm/[id]') ||
-    router.pathname.startsWith('/ecosystem') ||
-    router.pathname.startsWith('/discover')
-  ) {
-    return (
-      <HubApp>
-        <Component {...pageProps} />
-      </HubApp>
-    )
-  }
-
   return (
-    <BaseApp>
-      <Component {...pageProps} />
-    </BaseApp>
+    <QueryClientProvider client={queryClient}>
```

Comment:
As far as I can tell there is no reason that `HubApp` and `BaseApp` should not share a `QueryClientProvider`